state/api/src/main/java/com/expediagroup/streamplatform/streamregistry/state/model/specification/Tag.java

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright (C) 2018-2020 Expedia, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.expediagroup.streamplatform.streamregistry.state.model.specification;
+
+import lombok.NonNull;
+import lombok.Value;
+
+@Value
+public class Tag {
+  @NonNull String name;
+  @NonNull String value;
```

Comment:
I think potentially a Tag value should be nullable, for example an Entity could be tagged as being 'UNUSED'.

Reply:
I think you're right - it's nullable in the graphql api. I'll have another look